compiler/ztests/fork-from-and-pass.yaml

```diff
@@ -0,0 +1,13 @@
+zed: |
+  zq 'fork (=> from (file a.zson) => pass)' a.zson
```

Comment:
Aren't we getting rid of pass maybe change to `yield this`?

Reply:
I'd prefer to make the switch if and when we do drop pass.